### ISSUE TITLE
Standalone server : java.lang.IllegalArgumentException: The server version (v2.2) does not match the client (2.2)

### ISSUE THREAD

**Issue by Utopiah** [Genymobile/scrcpy#4433]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Ubuntu 23.04
 - scrcpy version: 2.2 from master
 - installation method: manual build
 - device model: Quest 3
 - Android version: 12

**Describe the bug**
Tried to follow https://github.com/Genymobile/scrcpy/blob/master/doc/develop.md#standalone-server in order to try with VLC but blocked by

```adb shell CLASSPATH=/data/local/tmp/scrcpy-server-manual.jar     app_process / com.genymobile.scrcpy.Server  2.2   tunnel_forward=true audio=false control=false cleanup=false     raw_stream=true max_size=1920

[server] ERROR: The server version (v2.2) does not match the client (2.2)
java.lang.IllegalArgumentException: The server version (v2.2) does not match the client (2.2)
        at com.genymobile.scrcpy.Options.parse(Options.java:237)
        at com.genymobile.scrcpy.Server.internalMain(Server.java:207)
        at com.genymobile.scrcpy.Server.main(Server.java:190)
        at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
        at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:355)

```

**Comment by rom1v** [comment]
Replace `v2.2` by `2.2`.

**Comment by Utopiah** [comment]
well that was silly of me, a type I imagine, anyway getting : 

```
fabien@fabien-CORSAIR-ONE-i160:~$ adb forward tcp:1234 localabstract:scrcpy
1234
fabien@fabien-CORSAIR-ONE-i160:~$ adb shell CLASSPATH=/data/local/tmp/scrcpy-server-manual.jar \
    app_process / com.genymobile.scrcpy.Server 2.2 \
    tunnel_forward=true audio=false control=false cleanup=false \
    raw_stream=true max_size=1920
Aborted 
```

**Comment by rom1v** [comment]
Oh, I wrongly passed `v2.2` instead of `2.2` as parameter of [bump_version](https://github.com/Genymobile/scrcpy/blob/master/bump_version): https://github.com/Genymobile/scrcpy/commit/c3c7bf7af329c8a5fd9f0310eae3b4e0a5f86207

This will be fixed when I will bump the numbers for the next release. Sorry for the inconvenience.

**Comment by Utopiah** [comment]
No worries, thanks for the clarification.

IMHO because it does work with other clients, e.g VLC, then this should be a non blocking warning rather than an error.

**Comment by rom1v** [comment]
Fixed in 2.3 :)
